src/main/scala/atom/config-schema/SettingsGroup.scala

```diff
@@ -3,16 +3,15 @@ package laughedelic.atom.config
 import scala.scalajs.js
 
 class SettingsGroup[S <: ConfigSchema](
+  schema: S,
   val title: js.UndefOr[String] = js.undefined,
-  val description: js.UndefOr[String] = js.undefined,
```

Comment:
Atom [API docs](https://atom.io/docs/api/v1.24.0/Config#title-and-description) say: